libs/designsystem/page/src/page.component.ts

```diff
@@ -210,7 +210,7 @@ export class PageComponent
   @Input() defaultBackHref: string;
   @Input() hideBackButton: boolean;
   @Input() titleMaxLines: number;
-  @Input() maxWidth: 'default' | 'standard' | 'optimized' | 'full' = 'default';
+  @Input() maxWidth: 'default' | 'optimized' | 'full' = 'default';
```

Comment:
Apply this to prevent breaking changes as the API will stay the same. However, the option will be redundant with default as those will result in the samme thing. 

```suggestion
  @Input() maxWidth: 'default' | 'standard' | 'optimized' | 'full' = 'default';
```

Reply:
I think it is better to prevent the breaking change and `console.log()` a deprecation warning whenever `standard` is used. And then remove `standard` in v10.